ipatests/prci_definitions/nightly_ipa-4-8.yaml

```diff
@@ -1472,3 +1472,15 @@ jobs:
         template: *ci-master-f30
         timeout: 7200
         topology: *adroot_adchild_adtree_master_1client
+
+  fedora-latest/test_backup_and_restore_TestBackupAndRestoreTrust:
```

Comment:
This should be `fedora-30` instead of `fedora-latest`.

Reply:
Updated.